### PR TITLE
Add styling of upload list on Welcome screen

### PR DESCRIPTION
**Pull request by igorkravchenko**
Add ability to change styles for upload list component and put file upload list related components into separate files.

MOB-1839

### CODE REVIEW

File: GliaWidgets/Asset.swift [hunk 1]
```diff
@@ -74,10 +75,11 @@ public enum Asset {
     public static let upgradeAudio = ImageAsset(name: "upgradeAudio")
     public static let upgradePhone = ImageAsset(name: "upgradePhone")
     public static let upgradeVideo = ImageAsset(name: "upgradeVideo")
+      public static let mcRemoveUpload = ImageAsset(name: "mc-remove-upload")
```
Comment (EgorovEI) [inline GliaWidgets/Asset.swift:78]: strange alignment

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift [hunk 2]
```diff
@@ -0,0 +1,232 @@
+#if canImport(QuickLookThumbnailing)
+    import QuickLookThumbnailing
+#endif
+
+import UIKit
+
+extension SecureConversations {
+    class FilePreviewView: UIView {
```
Comment (EgorovEI) [inline GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift:8]: Can be final?

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+            addSubview(imageView)
+            imageView.autoPinEdgesToSuperviewEdges()
+
+            addSubview(label)
+            label.autoPinEdge(toSuperviewEdge: .left, withInset: 5)
+            label.autoPinEdge(toSuperviewEdge: .right, withInset: 5)
+            label.autoAlignAxis(toSuperviewAxis: .horizontal)
+        }
+
+        func renderProps() {
```
Comment (EgorovEI) [inline GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift:64]: Can it be `private`?

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListView.swift [hunk 4]
```diff
@@ -0,0 +1,178 @@
+import UIKit
+
+extension SecureConversations {
+    class FileUploadListView: UIView {
```
Comment (EgorovEI) [inline GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListView.swift:4]: final?

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadView.swift [hunk 5]
```diff
@@ -0,0 +1,412 @@
+import UIKit
+
+extension SecureConversations {
+    class FileUploadView: UIView {
```
Comment (EgorovEI) [inline GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadView.swift:4]: final?

File: GliaWidgets/Sources/View/Chat/Entry/Upload/FileUploadListView.swift [hunk 6]
```diff
@@ -1,5 +1,7 @@
+// TODO: Remove this file once SecureConversations.FileUploadListView is fully integrsted.
```
Comment (EgorovEI) [inline GliaWidgets/Sources/View/Chat/Entry/Upload/FileUploadListView.swift:1]:
```suggestion
// TODO: Remove this file once SecureConversations.FileUploadListView is fully integrated.
```

**Comment by EgorovEI** [comment]
What do you think about to create tech-debt ticket for moving class-based styles to stucture-based? 
@igorkravchenko @dukhovnyi @rasmustautsglia

File: GliaWidgets/Asset.swift (hunk 1 above)
Comment (igorkravchenko) [reply GliaWidgets/Asset.swift:78]: Agree. But this is auto generated.

File: GliaWidgets/Sources/View/Chat/Entry/Upload/FileUploadListView.swift (hunk 6 above)
Comment (igorkravchenko) [reply GliaWidgets/Sources/View/Chat/Entry/Upload/FileUploadListView.swift]: Fixed 26bd52271dfcf47c510b4bbae624baa646c8fe43.

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadView.swift (hunk 5 above)
Comment (igorkravchenko) [reply GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadView.swift]: Added here 26bd52271dfcf47c510b4bbae624baa646c8fe43.

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListView.swift (hunk 4 above)
Comment (igorkravchenko) [reply GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListView.swift]: Added here 26bd52271dfcf47c510b4bbae624baa646c8fe43.

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift (hunk 3 above)
Comment (igorkravchenko) [reply GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift]: Added here 26bd52271dfcf47c510b4bbae624baa646c8fe43.

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift (hunk 2 above)
Comment (igorkravchenko) [reply GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift]: Added here 26bd52271dfcf47c510b4bbae624baa646c8fe43.

**Comment by igorkravchenko** [comment]
> What do you think about to create tech-debt ticket for moving class-based styles to stucture-based?
> @igorkravchenko @dukhovnyi @rasmustautsglia

We do not. This is also can be considered as breaking change since this is public facing API.

**Review by EgorovEI — APPROVED** [review]

**Review by dukhovnyi — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!merge

**Pull request merged**